packages/mesa/build.sh

```diff
@@ -15,21 +16,33 @@ TERMUX_PKG_REPLACES="libmesa"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --cmake-prefix-path $TERMUX_PREFIX
 -Dcpp_rtti=false
--Dgbm=enabled
+-Dgbm=disabled
 -Dopengl=true
 -Degl=enabled
 -Degl-native-platform=x11
--Dgles1=disabled
+-Dgles1=enabled
 -Dgles2=enabled
 -Ddri3=enabled
+-Ddri-drivers=[]
 -Dglx=dri
 -Dllvm=enabled
--Dshared-llvm=disabled
--Dplatforms=x11
--Dgallium-drivers=swrast
+-Dshared-llvm=enabled
```

Comment:
Is shared-llvm necessary here? This can cause `mesa` to be broken by major bump of `libllvm`.

Reply:
If it is absolutely necessary, then please include some version guard to avoid such a breakage:

https://github.com/termux/termux-packages/blob/a529efc1f1ad80cd4280c01f979b85c807cefd66/packages/rust/build.sh#L8-L10

Reply:
I relied on this https://github.com/termux/termux-packages/issues/10103#issue-1207999283